tests/pbh_input/assert_show_output.py

```diff
@@ -0,0 +1,78 @@
+"""
+Module holding the correct values for show CLI command outputs for the pbh_test.py
+"""
+
+show_pbh_hash_fields="""\
+NAME               FIELD              MASK       SEQUENCE    SYMMETRIC
+-----------------  -----------------  ---------  ----------  -----------
+inner_ip_proto     INNER_IP_PROTOCOL  N/A        1           No
+inner_l4_dst_port  INNER_L4_DST_PORT  N/A        2           Yes
+inner_l4_src_port  INNER_L4_SRC_PORT  N/A        2           Yes
+inner_dst_ipv4     INNER_DST_IPV4     255.0.0.0  3           Yes
+inner_src_ipv4     INNER_SRC_IPV4     0.0.0.255  3           Yes
+inner_dst_ipv6     INNER_DST_IPV6     ffff::     4           Yes
+inner_src_ipv6     INNER_SRC_IPV6     ::ffff     4           Yes
+"""
+
+
+show_pbh_hash="""\
+NAME           HASH FIELD
+-------------  -----------------
+inner_v4_hash  inner_ip_proto
+               inner_l4_dst_port
+               inner_l4_src_port
+               inner_dst_ipv4
+               inner_src_ipv4
+inner_v6_hash  inner_ip_proto
+               inner_l4_dst_port
+               inner_l4_src_port
+               inner_dst_ipv6
+               inner_src_ipv6
```

Comment:
If we won't have too many Hash Field Items / Name can maybe consolidate this into one row / Name

Reply:
@kktheballer the question is not clear: can you please share an example for such a use case

Reply:
What I meant was that if we have too many hash field items and we make each item go into each row we may have to scroll down to view all the output.

If we will only have 4-8 hash field items per inner_v4/6_hash we could leave it as is

Reply:
@kktheballer so far there will be only up to 7 hash fields. If we introduce more, than we can revisit the current CLI business logic. Are you ok with that?

Reply:
Sure that sounds good @nazariig